sync/test_sync.py

```diff
@@ -105,7 +104,7 @@ def test_yaml_files_to_dic_list(self):
         actual = None
         tmp_name = None
 
-        with tempfile.NamedTemporaryFile(delete=False) as tmp:
+        with tempfile.NamedTemporaryFile(dir='/tmp', delete=False) as tmp:
```

Comment:
Assuming `/tmp` is the temp folder is not portable, so I set the base dir explicitly to make the test run on mac.